src/check.ts

```diff
@@ -59,8 +59,17 @@ async function processSigs() {
         msgHash: true
       }
     };
-    const results = await snapshot.utils.subgraphRequest(subgraphUrl, query);
-    results.sigs.forEach(sig => processSig(sig.account, sig.msgHash));
+    
+    interface Results {
+      sigs?: [{ account: string; msgHash: string }];
+    }
```

Comment:
Can you just move this out of the function, otherwise it's good feel free to merge